README.md

```diff
@@ -29,6 +29,7 @@ spec:
         volumeMounts:
         - name: container-backup
           mountPath: /host
+      hostNetwork: true
```

Comment:
You could add backoffLimit to 0 with this PR

Reply:
Yes, I could. If I have a need to update this PR, I will include that.

Reply:
Done